project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module

```diff
@@ -46,7 +46,8 @@ function c4m_features_homepage_c4m_suggested_groups_get_groups() {
   $query
     ->entityCondition('entity_type', 'node')
     ->entityCondition('bundle', 'group')
-    ->propertyCondition('status', NODE_PUBLISHED);
+    ->propertyCondition('status', NODE_PUBLISHED)
+    ->fieldCondition('c4m_og_status', 'value', 'published');
```

Comment:
In the issue was requested to filter out also drafts, pendings and deleted
groups:
https://issuetracker.amplexor.com/jira/browse/CFM-1192?focusedCommentId=256047&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-256047

2016-09-07 13:24 GMT+02:00 ordavidil <notifications@github.com>:

> In project/profiles/capacity4more/modules/c4m/features/c4m_features_
> homepage/c4m_features_homepage.module
> <https://github.com/capacity4dev/capacity4more/pull/1281#discussion_r77803733>
> :
>
> > @@ -46,7 +46,8 @@ function c4m_features_homepage_c4m_suggested_groups_get_groups() {
> >    $query
> >      ->entityCondition('entity_type', 'node')
> >      ->entityCondition('bundle', 'group')
> > -    ->propertyCondition('status', NODE_PUBLISHED);
> > +    ->propertyCondition('status', NODE_PUBLISHED)
> > +    ->fieldCondition('c4m_og_status', 'value', 'published');
>
> yes, but others as well, why not adding a condition to filter out
> specifically the *archived* ones ?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/capacity4dev/capacity4more/pull/1281/files/5fd68667bddd54e8ae436c1264ae36648524ca39#r77803733>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA1hyrlTm5aG0z-rJuHVdjAVfPXABrprks5qnp78gaJpZM4J1KPh>
> .
>